low-level/sspl-ll.spec

```diff
@@ -107,17 +113,14 @@ mkdir -p /var/%{product_family}/sspl/bundle /var/log/%{product_family}/sspl /etc
 SSPL_DIR=/opt/seagate/%{product_family}/sspl
 
 [ -d "${SSPL_DIR}" ] && {
-    ln -sf $SSPL_DIR/low-level/framework/sspl_ll_d /usr/bin/sspl_ll_d
-    ln -sf $SSPL_DIR/low-level/framework/sspl_ll_d $SSPL_DIR/bin/sspl_ll_d
-    ln -sf $SSPL_DIR/low-level/files/opt/seagate/sspl/setup/generate_resource_health_view/resource_health_view /usr/bin/resource_health_view
-    ln -sf $SSPL_DIR/low-level/files/opt/seagate/sspl/setup/generate_sspl_bundle/sspl_bundle_generate /usr/bin/sspl_bundle_generate
-    ln -sf $SSPL_DIR/low-level/files/opt/seagate/sspl/setup/generate_sspl_bundle/sspl_bundle_generate $SSPL_DIR/bin/sspl_bundle_generate
-    ln -sf $SSPL_DIR/low-level/files/opt/seagate/sspl/setup/manifest_support_bundle /usr/bin/manifest_support_bundle
-    ln -sf $SSPL_DIR/low-level/framework/sspl_rabbitmq_reinit.py $SSPL_DIR/bin/sspl_rabbitmq_reinit
-    ln -sf $SSPL_DIR/low-level/framework/base/sspl_constants.py $SSPL_DIR/bin/sspl_constants.py
-    ln -sf $SSPL_DIR/low-level/framework/sspl_init $SSPL_DIR/bin/sspl_init
-    ln -sf $SSPL_DIR/low-level/framework/sspl_reinit $SSPL_DIR/bin/sspl_reinit
-    ln -sf $SSPL_DIR/low-level/framework/utils/salt_util.py $SSPL_DIR/bin/salt_util.py
+    ln -sf $SSPL_DIR/low-level/resource_health_view $SSPL_DIR/bin/resource_health_view
+    ln -sf $SSPL_DIR/low-level/sspl_bundle_generate $SSPL_DIR/bin/sspl_bundle_generate
+    ln -sf $SSPL_DIR/low-level/manifest_support_bundle $SSPL_DIR/bin/manifest_support_bundle
+    ln -sf $SSPL_DIR/low-level/sspl_setup $SSPL_DIR/bin/sspl_setup
```

Comment:
why items from here soft linked in /usr/bin

Reply:
It is /opt/seagate/sspl/bin only. we are copying that independent file to low-level to access SSPL code and then we are moving that to sspl/bin.
Please see the below comments:
https://github.com/Seagate/cortx-monitor/pull/304#discussion_r573594603

Reply:
i get it